storm-server/src/main/java/org/apache/storm/daemon/supervisor/BasicContainer.java

```diff
@@ -346,7 +346,7 @@ protected String getWildcardDir(File dir) {
     }
     
     protected List<String> frameworkClasspath(SimpleVersion topoVersion) {
-        File stormWorkerLibDir = new File(_stormHome, "lib-worker");
+        File stormWorkerLibDir = new File(_stormHome, "lib");
```

Comment:
This was an effort to reduce dependencies on worker. If the line makes issue, let's fix it. Which issues you are seeing before fixing this?